napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -71,7 +72,7 @@ def __init__(
         object_name: str = '',
         add_vertical_stretch=True,
     ):
-        self.qt_viewer = qt_viewer
+        self.ref_qt_viewer = ref(qt_viewer)
```

Comment:
should we make this private?

Reply:
my goal is maybe to replace `qt_viewer`, but yes I can try to make it private.